content/2019/06/17/refactrtech.rst

```diff
@@ -0,0 +1,168 @@
+public: yes
+author: erica
+tags: [Conferences, JavaScript, Diversity]
+image:
+  - src: 'blog/2019/refactr-tech-panel.jpg'
+headline:
+  - tagline: |
+      Refactr.tech 2019 aims to solve tech's diversity problem
+
+summary: |
+  As we readied our seats for the first ever `Refactr.tech Conference`_, I
+  pondered the question: What would a technical conference look like if its
+  organizers were intentional about inclusiveness but still centered on tech?
+
+  .. _Refactr.tech Conference: http://refactr.tech/
+
+
+Diversity. Inclusion. Tech.
+===========================
+
+Why this matters.
+-----------------
+
+For three days, technologists from many parts of the country gathered in Atlanta
+GA – a melting pot of human experience in itself – to realize just that.
+`Refactr.tech`_ showcased growing and powerful voices of underrepresented
+professionals in tech and their allies; with a focus on technology and creating
+a safe space to have conversations around diversity, inclusion, and
+intersectionality in tech.
+
+  "I knew I was in the right place as I met so many people I already follow on
```

Comment:
This should probably get an attribution, or use the pullquote macro instead: https://www.oddbird.net/docs/sample/#blockquotes-and-pullquotes

Reply:
@mirisuzanne I'm actually just quoting myself and just wanted to draw attention to the sentence 🤷🏽, but made changed to a pull-quote, without attributing myself.  Let me know if that's still appropriate.